docs/testplan/DHCP-relay-stress-test.md

```diff
@@ -0,0 +1,73 @@
+# DHCP Relay Stress Test Plan
+- [Overview](#overview)
+- [Scope](#scope)
+- [Scale / Performance](#scale--performance)
+- [Related DUT CLI commands](#related-dut-cli-commands)
+- [Test structure](#test-structure)
+  - [Configuration](#configuration)
+  - [Test cases](#test-cases)
+    - [Test case](#test-case)
+
+
+
+## Overview
+
+The purpose is to test the DHCP relay service can survive the max load that we lift to CPU.
+
+### Scope
+---------
+
+CoPP is to limit the rate of traffic sent to CPU, and then generating packets above this maximum rate, the test is to ensure the DHCP relay service can handle the maximum load without failure.
+
+### Scale / Performance
+-------------------
+1. Verfy DHCP Container Status.
+2. CPU should remain within acceptable range, indicating the DUT is not overwhelmed.
+3. Check whether the function is still normal.
```

Comment:
Could you please share the detail of checking dhcp_relay functionality? My suggestion is to verify whether DHCP packets are relayed immediately or with little delay